src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1174,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
+     * @param fullString  Parameter as a string
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String sign) {
+        return fullString.replace(sign, "");
```

Comment:
as mentioned in class, replace also will replace other instances of the sign argument found in fullString.